packages/client/src/Auth.re

```diff
@@ -0,0 +1,43 @@
+type loginDetails = {
+  username: string,
+  password: string,
+};
+
+module Payload = {
```

Comment:
Is `payload` what its named in the API? I wonder if we could do something more declarative? 🤔 

Also, I think it makes sense to put all these in `Shared.re`!

Reply:
The graphql return type for a login or register mutation is AuthPayload. The thinking here was that this module gives us access to Auth.Payload. We can definitely move it to shared and give it a more declarative name.